jmh.gradle

```diff
@@ -52,10 +52,10 @@
 
 sourceSets {
     jmh {
-	java.srcDir 'src/jmh/java'
-	resources.srcDir 'src/jmh/resources'
-	compileClasspath += main.output
-	runtimeClasspath += main.output
+        java.srcDir 'src/jmh/java'
```

Comment:
any idea why all these reformats happened?

Reply:
If these are expected reformats consider to send a quick PR to do only the reformat and will merge that asap.

Reply:
yeap i reformat it according to others *.gradle files and also change deprecated method, will do in another PR as suggested